src/iuse_actor.cpp

```diff
@@ -1566,6 +1566,44 @@ long fireweapon_on_actor::use( player *p, item *it, bool t, const tripoint& ) co
     return it->type->charges_to_use();
 }
 
+void manualnoise_actor::load( JsonObject &obj )
+{
+    use_message         = obj.get_string( "use_message" );
+    noise_message       = obj.get_string( "noise_message", "" );
+    noise               = obj.get_int( "noise", 0 );
+    moves               = obj.get_int( "moves", 0 );
+}
+
+iuse_actor *manualnoise_actor::clone() const
+{
+    return new manualnoise_actor( *this );
+}
+
+long manualnoise_actor::use( player *p, item *it, bool t, const tripoint& ) const
+{
+    if( t ) {
+        return 0;
+    }
+
+    p->moves -= moves;
+    {
```

Comment:
Why the `{` here?
You could have a charge check here, though. And move the `p->moves` thing into the braces.